packages/dso-toolkit/src/components/accordion/accordion.args.ts

```diff
@@ -14,7 +14,7 @@ import { LabelStatus } from "../label";
 import { RenvooiValue } from "../renvooi";
 
 export interface AccordionArgs {
-  variant: undefined | "compact" | "conclusion" | "neutral" | "compact-black";
+  variant: "default" | "compact" | "conclusion" | "neutral" | "compact-black";
```

Comment:
Tsja, ik begrijp het... Maar ik ben het niet helemaal eens met de oplossing. Want de bug zit niet in Storybook, maar in ons component. Het component gaat niet goed om met `variant = undefined`.

Zie ook mijn comment in het issue https://github.com/dso-toolkit/dso-toolkit/issues/2957#issuecomment-2631412835

Eigenlijk verwacht ik deze change:

```suggestion
  variant: undefined | "default" | "compact" | "conclusion" | "neutral" | "compact-black";
```